.github/workflows/npm_audit_chatbot.yml

```diff
@@ -0,0 +1,43 @@
+name: Chatbot - npm-audit
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - ansible_ai_connect_chatbot/**
+  pull_request:
+    branches:
+      - main
+    paths:
+      - ansible_ai_connect_chatbot/**
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    env:
+      WORKING_DIRECTORY: ./ansible_ai_connect_chatbot
```

Comment:
wow is this the chatbot UI path? Why we did not include a suffix as _ui ? (just a related question)

Reply:
I think the name `ansible_ai_connect_chatbot` is sufficient since we know we implement serivice-related logic in the Python/Django code under `ansible_ai_connect`.

Reply:
not enough for "clean code" to me, but fair enough, thanks!

Reply:
Thanks!